ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_bgp.schema.yml

```diff
@@ -2147,6 +2162,12 @@ keys:
               description: VRF name.
               convert_types:
                 - int
+            bgp:
+              type: dict
+              keys:
+                redistribute_internal:
+                  type: bool
+                  description: Allow redistribution of iBGP routes into an Interior Gateway Protocol (IGP). EOS default is true.
```

Comment:
I think same description is applicable for all places

Reply:
Fixed.